tools/chromiumos/gen_autosuspend_rules.py

```diff
@@ -1,10 +1,7 @@
 #!/usr/bin/env python3
-# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
you need to manually drop these changes to the header in this file before committing

Reply:
why?

Reply:
Because it drops the SPDX line and refers to a license file that doesn't exist

Reply:
fixed it myself so we can merge this